dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/JdbcEventStore.java

```diff
@@ -1274,7 +1275,9 @@ private String getCategoryOptionComboQuery(User user) {
     String joinCondition =
         "inner join categoryoptioncombo coc on coc.categoryoptioncomboid = ev.attributeoptioncomboid "
             + " inner join (select coc.categoryoptioncomboid as id,"
-            + " string_agg(co.uid, ';') as co_uids, count(co.categoryoptionid) as co_count"
+            + " string_agg(co.uid, '"
+            + TextUtils.COMMA
```

Comment:
I would go for the raw string

```java
    String joinCondition =
        """
inner join categoryoptioncombo coc on coc.categoryoptioncomboid = ev.attributeoptioncomboid
inner join
  (select coc.categoryoptioncomboid as id, string_agg(co.uid, ';') as co_uids, count(co.categoryoptionid) as co_count
  from categoryoptioncombo coc
  inner join categoryoptioncombos_categoryoptions cocco on coc.categoryoptioncomboid = cocco.categoryoptioncomboid
  inner join dataelementcategoryoption co on cocco.categoryoptionid = co.categoryoptionid
  group by coc.categoryoptioncomboid
""";
```

which makes the SQL readable. I find the `+` we already have + now using a constant for the `,` makes the SQL hard to read.